src/nvim/api/vim.c

```diff
@@ -987,6 +987,78 @@ Buffer nvim_create_buf(Boolean listed, Boolean scratch, Error *err)
   return buf->b_fnum;
 }
 
+/// Open a new window.
+///
+/// Currently this is used to open floating and external windows.
+/// Floats are windows that are drawn above the split layout, at some anchor
+/// position in some other window. Floats can be draw internally or by external
+/// GUI with the |ui-multigrid| extension. External windows are only supported
+/// with multigrid GUIs, and are displayed as separate top-level windows.
+///
+/// Exactly one of `external` and `relative` must be specified.
+///
+/// @param buffer handle of buffer to be displayed in the window
+/// @param enter whether the window should be entered (made the current window)
```

Comment:
Vim parlance might be `current` (thinking of `NormalNC` etc.). Or else `focus` (in analogy with `FocusGained/Lost`), which is also more common.

Reply:
focus as opposed to _what_? "current" is our jargon for current window (`nvim_get/set_current_xx`). "enter" is our jargon for entering a window/buffer (`BufEnter` / `WinEnter`).